Lib/test/test_socket.py

```diff
@@ -6048,9 +6048,67 @@ def test_new_tcp_flags(self):
         self.assertEqual([], unknown,
             "New TCP flags were discovered. See bpo-32394 for more information")
 
+class CreateServerSockTest(unittest.TestCase):
+
+    def echo_server(self, sock, port=None):
+        def run():
+            with sock:
+                conn, _ = sock.accept()
+                with conn:
+                    msg = conn.recv(1024)
+                    if not msg:
+                        return
+                    conn.sendall(msg)
+
+        sock.settimeout(2)
+        t = threading.Thread(target=run)
+        t.start()
+        time.sleep(.1)
```

Comment:
Not sure this is reliable. @vstinner will probably object.

Reply:
I object. Don't do that.